packages/cozy-client/src/models/geo.js

```diff
@@ -16,6 +19,15 @@ const degreesToRadians = degrees => {
   return (degrees * Math.PI) / 180
 }
 
+const radiansToDegrees = radians => {
+  return (radians * 180) / Math.PI
+}
+
+const roundToNDecimals = (value, n) => {
+  const multiplier = Math.pow(10, n)
+  return Math.round(value * multiplier) / multiplier
+}
```

Comment:
let's add the jsdoc for both of the new methods! (also you can add // ts-check on at the top)

Reply:
Fair enough, I didn't do it as it was small internal methods

Reply:
[c7d07ce](https://github.com/cozy/cozy-client/pull/1397/commits/c7d07ce4d8ab58d6ac6964615121a763f48e2818)